pkgs/applications/audio/ardour/default.nix

```
@@ -147,18 +152,25 @@ stdenv.mkDerivation rec {
       --replace 'ffprobe_exe = X_("");' 'ffprobe_exe = X_("${ffmpeg_3}/bin/ffprobe");'
   '';
 
-  postInstall = ''
-    # wscript does not install these for some reason
-    install -vDm 644 "build/gtk2_ardour/ardour.xml" \
-      -t "$out/share/mime/packages"
-    install -vDm 644 "build/gtk2_ardour/ardour6.desktop" \
-      -t "$out/share/applications"
-    for size in 16 22 32 48 256 512; do
-      install -vDm 644 "gtk2_ardour/resources/Ardour-icon_''${size}px.png" \
-        "$out/share/icons/hicolor/''${size}x''${size}/apps/ardour6.png"
-    done
-    install -vDm 644 "ardour.1"* -t "$out/share/man/man1"
-  '';
+  postInstall = let
+    script = ''
+      # wscript does not install these for some reason
+      install -vDm 644 "build/gtk2_ardour/ardour.xml" \
+        -t "$out/share/mime/packages"
+      install -vDm 644 "build/gtk2_ardour/ardour6.desktop" \
+        -t "$out/share/applications"
+      for size in 16 22 32 48 256 512; do
+        install -vDm 644 "gtk2_ardour/resources/Ardour-icon_''${size}px.png" \
+          "$out/share/icons/hicolor/''${size}x''${size}/apps/ardour6.png"
+      done
+      install -vDm 644 "ardour.1"* -t "$out/share/man/man1"
+    '';
```

```suggestion
    '' + lib.optionalString ''
      videoSupportScript = lib.optionalString videoSupport ''
      # `harvid` and `xjadeo` must be accessible in `PATH` for video to work.
      wrapProgram "$out/bin/ardour6" \
        --prefix PATH : "${lib.makeBinPath [ harvid xjadeo ]}"
    '';
```
This is far simpler than concatStrings.

---

Thanks! I'd forgotten you could concatenate strings with `+`.